test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -54,16 +58,16 @@ namespace iroha {
       }
 
       void SetUp() override {
-        ASSERT_FALSE(boost::filesystem::exists(block_store_path))
-            << "Temporary block store " << block_store_path
-            << " directory already exists";
-        connect();
+        if (not initialised) {
```

Comment:
Move this code to constructor if it has to be run only once?

Reply:
Since asserts cannot be moved into constructor they are stayed in a setup method to keep all of the previous check logic